### DISCUSSION TITLE
Helix not playing nice with jdtls

### DISCUSSION THREAD

**Discussion by shinysocks** [helix-editor/helix#9446] [Troubleshooting] [answered]
I am unable to get jdtls to work for the helix editor, has anyone got this working?? what is your configuration for helix?

**Comment by shinysocks** [comment, marked as answer]
sorry, I just got this working after I submitted the issue. I'm on debian-based so jdtls is not packaged to clarify
- I used this amazing script: [jdtls-launcher](https://github.com/eruizc-dev/jdtls-launcher)
- my `~/.config/helix/languages.toml` config looks like this:
```
[language-server]
jdtls = { command = "jdtls" }
language-server = { command = "jdtls", args = [ 
  "-configuration", "/home/shinysocks/.cache/jdtls/config",
  "-data", "/home/shinysocks/.cache/jdtls/workspace",
  ]}

[[language]]
name = "java"
scope = "source.java"
injection-regex = "java"
file-types = ["java"]
roots = ["pom.xml", "build.gradle", ]
indent = { tab-width = 4, unit = "    " }
language-servers = [ "jdtls" ]

[[grammar]]
name = "java"
source = { git = "https://github.com/tree-sitter/tree-sitter-java", rev = "09d650def6cdf7f479f4b78f595e9ef5b58ce31e" }
```
- it would be awesome if this script could be mentioned on the page with language info so it's easier for beginners to find.

**Reply by gabydd** [reply]
Feel free to add it to the wiki

**Reply by shinysocks** [reply]
ah yeah, will do
